public/css/eventList.css

```diff
@@ -0,0 +1,80 @@
+.upper span {
+    display: block;
+    text-align: center;
+}
+
+.mid span {
+    display: block;
+    text-align: center;
+}
+
+#eventTitle {
+    text-align: center;
+}
+
+.event {
```

Comment:
Is clickable, so could add `cursor: pointer`.

Reply:
nice

Reply:
e120b14